### DISCUSSION TITLE
/photoprism/storage/config/settings.yml   no such file or directory)

### DISCUSSION THREAD

**Discussion by gerroon** [photoprism/photoprism#3462] [Q&A]
Hi

I am using docker-compose to run it. I believe I have it set correctly but for some reason it cant find this file in the container, so I cant access the web page

The things is "settings.yml" exists in the docker mapped volume.
```
ls
certificates  hub.yml  settings.yml
```

``` level=error msg="settings: could not create /photoprism/storage/config/settings.yml (open /photoprism/storage/config/settings.yml: no such file or directory)"    ```

**Comment by lastzero** [comment]
This file is created automatically to store the UI settings. Could it be that the permissions are not set correctly or there is some other issue with the volume?
